### PR TITLE
torbrowser-launcher: hardening

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/profile-m-z/torbrowser-launcher.profile [hunk 1]
```diff
@@ -53,12 +53,14 @@ nou2f
...
 protocol unix,inet,inet6
 seccomp !chroot
+seccomp.block-secondary
 #tracelog - may cause issues, see #1930
 
 disable-mnt
 private-bin bash,cat,cp,cut,dirname,env,expr,file,gpg,grep,gxmessage,id,kdialog,ln,mkdir,mv,python*,rm,sed,sh,tail,tar,tclsh,test,tor-browser,tor-browser-en,torbrowser-launcher,update-desktop-database,xmessage,xz,zenity
 private-dev
 private-etc @tls-ca
+private-opt tor-browser
```
Comment (rusty-snake) [inline etc/profile-m-z/torbrowser-launcher.profile:63]: Can cause slow startup.

Breaks if any other name is used (e.g. torbrowser-launcher).

File: etc/profile-m-z/torbrowser-launcher.profile [hunk 2]
```diff
@@ -53,12 +53,14 @@ nou2f
...
 private-tmp
 
 dbus-user none
```
Comment (rusty-snake) [inline etc/profile-m-z/torbrowser-launcher.profile:66]: Some namespaces can be restricted.

File: etc/profile-m-z/torbrowser-launcher.profile [hunk 3]
```diff
@@ -33,9 +33,9 @@ whitelist ${HOME}/.local/share/torbrowser
 whitelist /opt/tor-browser
 whitelist /usr/share/torbrowser-launcher
 include whitelist-common.inc
-include whitelist-var-common.inc
 include whitelist-runuser-common.inc
 include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/torbrowser-launcher.profile:38]: Maybe we can include whitelist run.

File: etc/profile-m-z/torbrowser-launcher.profile [hunk 4]
```diff
@@ -33,9 +33,9 @@ whitelist ${HOME}/.local/share/torbrowser
 whitelist /opt/tor-browser
 whitelist /usr/share/torbrowser-launcher
```
Comment (rusty-snake) [inline etc/profile-m-z/torbrowser-launcher.profile:34]: Maybe we can include disable process.

**Review by rusty-snake — COMMENTED** [review]
More hardening possible if you want. Here and on mullvad.

File: etc/profile-m-z/torbrowser-launcher.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-m-z/torbrowser-launcher.profile:63]: Better revert it in that case. Thanks for the info!

File: etc/profile-m-z/torbrowser-launcher.profile (hunk 3 above)
Comment (glitsj16) [reply etc/profile-m-z/torbrowser-launcher.profile:38]: Tested, we can add it.

File: etc/profile-m-z/torbrowser-launcher.profile (hunk 4 above)
Comment (glitsj16) [reply etc/profile-m-z/torbrowser-launcher.profile:35]: Doesn't seem to cause any issues. Let's add it!

**Comment by glitsj16** [comment]
@kmk3 @rusty-snake If you find the time, can you have another look at this? I think it's okay now. Same for #5887.

File: etc/profile-m-z/torbrowser-launcher.profile [hunk 5]
```diff
@@ -53,15 +55,17 @@ nou2f
...
 private-dev
 private-etc @tls-ca
+#private-opt tor-browser - can cause slow startup
 private-tmp
 
 dbus-user none
 dbus-system none
 
-#restrict-namespaces
+restrict-namespaces
```
Comment (rusty-snake) [inline etc/profile-m-z/torbrowser-launcher.profile:71]: Weakens torbrowsers sandbox?

**Review by kmk3 — APPROVED** [review]

**Pull request merged**